.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -81,6 +81,9 @@ jobs:
       echo "##vso[task.setvariable variable=windowsAMAUrl;isOutput=true]$windowsAMAUrl"
     name: setup
 
+  - task: CredScan@3
```

Comment:
shouldn't this be run before "git push" since our repo is public? if it runs after git push the code is already publicly available

Reply:
yeah true. I added credscan that is built my msft. we will have to find something which github uses as well